otherlibs/runtime_events/runtime_events.ml

```diff
@@ -221,6 +221,11 @@ module Timestamp = struct
 
   let to_int64 t =
     t
+
+  external current_timestamp : unit -> (int64 [@unboxed]) =
```

Comment:
Minor nitpick, this would be called as `Timestamp.current_timestamp ()`? Wouldn't `Timestamp.current ()` or `Timestamp.get_current ()` read better?

Reply:
fair enough, `get_current` seems less redundant.